src/components/PlugMe.js

```diff
@@ -5,56 +5,95 @@ class PlugMe extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            club : {
+
                 points: 0,
                 name:"Man United",
                 color: "red",
                 gamesPlayed: 25,
                 currentStanding: 4
-            }
+
         };
+        // this.state = {
+        //             points: 0,
+        //             name:"Man United",
+        //             color: "red",
+        //             gamesPlayed: 25,
+        //             currentStanding: 4
+        //
+        //     };
         /**
          * This is manual binding in-case we want to define a function like this  >>> handlePoints () {}
          */
 
         // this.handlePoints = this.handlePoints.bind(this)
     }
     handlePoints=()=>{
-        this.setState({
-                                club: {points : 150}
-                            }
-                     );
+        // this.setState({
+        //                         club: {points : 150}
+        //                     }
+        //              );
+
+        //DOD:don't use setState
+
+        this.setState(prveState => {return{
+                    points:prveState.points = 150
+
+              }
+
+            })
+
+
     };
-    // handlePoints(){
-    //
-    // }
-    static getDerivedStateFromProps(props, state) {
-        /**
-         * Enable this to see the state being changed and also dont use setState Method by react
-         * Note: this is a life cycle method that will be called on component mounting and will return state
-         */
-        // return {
-        //     club : {points: 57}
-        // }
 
-        //TODO : Implement this in a way where you will not affect the rest of the state values
+    inputChangeHandler=(event)=>{
+        this.setState({
+                        [event.target.name]:event.target.value
```

Comment:
Don't use this for now. Use event handler for each input element separately.

Reply:
should I use 6 event handler ?